initializr-generator/src/main/java/io/spring/initializr/generator/buildsystem/maven/MavenBuildWriter.java

```diff
@@ -485,7 +486,7 @@ private void writeDeploymentRepository(IndentingWriter writer, String name, Depl
 	private void writeSingleElement(IndentingWriter writer, String name, String text) {
 		if (text != null) {
 			writer.print(String.format("<%s>", name));
-			writer.print(text);
+			writer.print(StringEscapeUtils.escapeXml10(text));
```

Comment:
I'd rather not bring an entire library at that level to only use one method. `initializr-generator` is pretty barebone and I'd like to keep it that way.

Reply:
Sure. I could work on that. Before I proceed, I've two questions:
1. What should be the scope of our symbols? i.e. Are we only limited to escaping these symbols: ", <, >, & and '?
2. If the answer to the above question is no, then how should we be dealing with other symbols that may appear like (⌊, ⌋, √, Δ)? The only reason why I went with the library was that it dealt with such cases.

Reply:
I'd escape regular XML entities for a start.

Reply:
Could you please have a look at it now?